extensions/amp-bind/amp-bind.md

```diff
@@ -89,30 +98,25 @@ In this example, `myState.foo` will evaluate to `"bar"`.
 - For security reasons, binding to `innerHTML` is disallowed
 - All attribute bindings are sanitized for unsafe URL protocols (e.g. `javascript:`)
 
-### Bindable attributes
+#### Element-specific attributes
 
-For non-AMP elements, most attributes accepted by the [AMP Validator](https://validator.ampproject.org/) are bindable.
+Most attributes accepted by the [AMP Validator](https://validator.ampproject.org/), for AMP and non-AMP elements, are bindable.
```

Comment:
TBD in #8056. We probably won't allow mutation of things like SVG and other elements that weren't in the (now removed) `BindValidator` rule spec.

Removed the comma.